src/Commands/SiteInstallCommands.php

```diff
@@ -29,14 +30,19 @@ public function siteInstallPostCommand($result, CommandData $commandData) {
       $result = \Drupal::service('acquia_cms_common.utility')->rebuildSiteStudio();
       $this->yell('Finished rebuilding.');
       $this->setFinishedTime();
-      return is_array($result) && isset(array_shift($result)['error']) ? CommandResult::exitCode(self::EXIT_FAILURE) : CommandResult::exitCode(self::EXIT_SUCCESS);
     }
+    // Send data to telemetry based upon certain conditions.
+    if (\Drupal::service('module_handler')->moduleExists('acquia_telemetry') && Environment::isAhEnv()) {
+      acquia_cms_send_heartbeat_event();
```

Comment:
@panshulK Please add a condition for function check. i.e
```suggestion
if (function_exists('acquia_cms_send_heartbeat_event')) {
  acquia_cms_send_heartbeat_event();
}
```

Reply:
Done, @vishalkhode1